.gitignore

```diff
@@ -8,4 +8,6 @@ content/commands/*
 content/topics/*
 !content/topics/_index.md
 _site
-_data/groups.json
\ No newline at end of file
+_data/groups.json
```

Comment:
Looks like this was a formatting change or something? Please note that so it doesn't look strange in the commit history.

Reply:
There was a newline missing at the end of the file. That's why GitHub shows this strange sign. git diff shows this as:

```diff
diff --git a/.gitignore b/.gitignore
index 4eac8e1..728ce5c 100644
--- a/.gitignore
+++ b/.gitignore
@@ -8,4 +8,6 @@ content/commands/*
 content/topics/*
 !content/topics/_index.md
 _site
-_data/groups.json
\ No newline at end of file
+_data/groups.json
+_data/resp2_replies.json
+_data/resp3_replies.json
```